scripts/install-deps.sh

```diff
@@ -142,6 +142,32 @@ fetch_and_unpack_range()
         https://github.com/ericniebler/range-v3/archive/refs/tags/0.11.0.tar.gz
 }
 
+fetch_and_unpack_libutempter()
+{
+    fetch_and_unpack \
+        libutempter-1.2.1 \
+        libutempter-1.2.1.tar.gz \
+        http://ftp.altlinux.org/pub/people/ldv/utempter/libutempter-1.2.1.tar.gz
+    printf \
+        "cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+project(utempter VERSION "1.2.1" LANGUAGES C)
+
+add_library(utempter
```

Comment:
```suggestion
add_library(utempter STATIC
```

`add_library` should mention `STATIC` as second parameter, to mandate linkage to be static. this way we ensure that this library is literally embedded into the target executable, also, not accidentally making it a DSO.